File: Modulo_I/banco/JAVA/ContaInterface.java
import java.util.Scanner;

class ContaInterface {

    public ContaInterface(){}

    public void Menu(Conta conta_cliente){

        Scanner teclado = new Scanner(System.in);

        while (true) { 

            System.out.println("---Menu---");
            System.out.println("Digite o número referente a ação: ");
            System.out.println("01: Saldo");
            System.out.println("02: Depositar");
            System.out.println("03: Sacar");
            System.out.println("04: Sair");
            
            int acao = teclado.nextInt();      

            switch (acao){

                case 1 -> System.out.println("R$ "+conta_cliente.conferir());

                case 2 -> {

                    System.out.println("Digite a quantidade de dinheiro que gostaria de depositar: ");
                    double deposito = teclado.nextDouble();
                    conta_cliente.depositar(deposito);
                }
                    
                case 3 -> {

                    System.out.println("Digite a quantidade de dinheiro que gostaria de sacar:");
                    double saque = teclado.nextDouble();
                    conta_cliente.sacar(saque);
                }

                case 4 -> {

                    return; 
                }

                default -> System.out.println("Opção inválida");
                
            }

        }
    }
}

File: Modulo_I/conta/PY/contaInterface.py
from conta import Conta

class ContaInterface:

    def menu(self, conta_cliente):

        while True:

            print('---Menu---')
            print('Digite o número referente a ação:')
            print('01: Saldo')
            print('02: Depositar')
            print('03: Sacar')
            print('04: Sair')

            acao = int(input())

            match acao:

                case 1:
                    print('R$', conta_cliente.conferir())

                case 2:
                    print('Digite a quantidade de dinheiro que gostaria de depositar:')
                    dinheiro = float(input())
                    conta_cliente.depositar(dinheiro)

                case 3:
                    print('Digite a quantidade de dinheiro que gostaria de sacar:')
                    dinheiro = float(input())
                    conta_cliente.sacar(dinheiro)

                case 4:
                    break

                case _:
                    print('Não há ações referente a esse número')